src/client.rs

```diff
@@ -257,6 +244,18 @@ pub struct ClientStats {
     pub indexes: HashMap<String, IndexStats>,
 }
 
+/// Health of the MeiliSearch server.
+/// Example:
+/// ```text
```

Comment:
It's not text but Rust code. `rust` is the default so we can omit the identifier of the language. 

```suggestion
/// ```
```